### DISCUSSION TITLE
I can not find two-stream i3d models. It has only rgb config files.

### DISCUSSION THREAD

**Discussion by mnnsolanki** [open-mmlab/mmaction2#1554] [Q&A] [answered]
As mentioned in the paper, we can pass images and optical flow in i3d architecture.

![image](https://user-images.githubusercontent.com/77401469/161094965-d3934e48-a47f-47da-b40e-670c385344d1.png)

I want to pass both image modalities (RGB + OF) while training(in figure - e). But currently, i3d configuration only support RGB images in MMAction2. 
Is there any way to use both modalities in i3d or any other network which accepts both?

**Comment by dreamerlin** [comment, marked as answer]
Currently, we haven't provided i3d with flow ckpts

**Reply by Hobeom** [reply]
https://github.com/open-mmlab/mmaction/blob/master/MODEL_ZOO.md
It seems that mmaction provided weights from DeepMind.
Is i3d architecture in DeepMind that can be brought in mmaction2?
